ambassador/ambassador/cli.py

```diff
@@ -199,19 +199,20 @@ def dump(config_dir_path: Parameter.REQUIRED, *,
         sys.exit(1)
 
 
-def validate(config_dir_path: Parameter.REQUIRED, *, k8s=False):
+def validate(config_dir_path: Parameter.REQUIRED, **kwargs):
     """
-    Validate an Ambassador configuration
+    Validate an Ambassador configuration. This is an extension of "config" that
+    redirects output to devnull and always exits on error.
 
     :param config_dir_path: Configuration directory to scan for Ambassador YAML files
     :param k8s: If set, assume configuration files are annotated K8s manifests
     """
-    config(config_dir_path, os.devnull, k8s=k8s, exit_on_error=True)
+    config(config_dir_path, os.devnull, exit_on_error=True, **kwargs)
```

Comment:
If `kwargs` also has `exit_on_error` set, this will be an error. Can that ever happen? (I don't think so.)

Reply:
Nah, it can't happen since `clize` doesn't pick up on `exit_on_error` since the default value is `True`, as far as I am aware at least.

Trying to specify it just results in an "Unknown option" error from what I can see.

Reply:
Good enough, thanks!